gcc/d/d-lang.cc

```diff
@@ -1079,6 +1079,7 @@ d_parse_file()
 	{
 	  File deps (global.params.moduleDepsFile);
 	  deps.setbuffer ((void *) ob->data, ob->offset);
+	  deps.ref = 1;
```

Comment:
Yes. It's very straight-forward, you can just look at other entries for the general flow.

Just needs to be a (dull and) concise description of the change without any rationale (reasoning, bug IDs, etc, is left for the commit message).

You can just copy and paste the following.
```
* d-lang.cc (d_parse_file): Set ref flag on the module and make deps
file handle.
```